pkg/core/localize/locales/en/cmd/rule.en.toml

```diff
@@ -182,38 +181,38 @@ one='Disabling "{{.RuleType}}" rule for artifact with ID "{{.ArtifactID}}"'
 one='Disabling all rules for artifact with ID "{{.ArtifactID}}"'
 
 [registry.rule.disable.flag.yes]
-one = 'Disable rule(s) without prompt'
```

Comment:
I tried to disable 2 enabled rules in one command, but only one rule was disabled. I wasn't sure if that was working as designed or a bug -- my apologies, I meant to highlight that question to you. If it should be possible to disable more than one rule at once, we can rephrase as "Disable one or more rules without prompt".

Reply:
Apologies for that, the CLI misses examples to disable both rules at once. Thanks for pointing it out :) . Will update it in this PR.
It can be done by not passing the `--rule-type` flag.
```
rhoas service-registry rule disable
```
"Disable one or more rules without prompt" looks good.

Reply:
We can update the text in the follow-up PR, when you add the example to disable both at once? Or feel free to update in this PR if you prefer :)

Reply:
We can do it in the follow up. :)

Reply:
Thanks @rkpattnaik780! Any additional changes to be made in this PR, or are you ready to approve now?